test/anacondalib.py

```diff
@@ -60,6 +77,30 @@ def setUp(self):
             self.allow_browser_errors(".*client closed.*")
             self.allow_browser_errors(".*Server has closed the connection.*")
 
+    def _execute(self, cmd):
+        return subprocess.check_call(cmd, stderr=subprocess.STDOUT, shell=True)
```

Comment:
I recommend to *not* redirect stderr. It's much more useful to see the error messages in the test log if this fails, than them getting hidden (as nothing happens with the return value).

Also, all calls to _execute() are a single program with arguments, no actual shell constructs. So dropping `shell` and passing an array is cleaner. It's ok for a test, just something to consider.

Reply:
Replaced as suggested.